### DISCUSSION TITLE
[META] Use #lark-parser tag for GitHub

### DISCUSSION THREAD

**Discussion by ThatXliner** [lark-parser/lark#842] [Ideas]
Since #lark is [quite used](https://github.com/topics/lark) and most of the results are unrelated. Use [#lark-parser](https://github.com/topics/lark-parser) (similar problem like #505 but this is a GitHub-wide issue) instead (currently, all the results are related and relevant).

Also, you should add the #lark-parser tag to this repo.

**Comment by erezsh** [comment]
Use it where? I don't understand.

How would adding this tag to the repo help?

**Reply by ThatXliner** [reply]
Here:
<img width="343" alt="Screen Shot 2021-02-22 at 11 01 19 AM" src="https://user-images.githubusercontent.com/66848002/108756392-4c43ed00-74fd-11eb-855d-dedfa19a5d0e.png">

**Comment by MegaIng** [comment]
I mean if you look through the results, this repo is the top result, and quite a few of the other results are related to this repo. I would much rather question why so many others are using lark as well. It doesn't seem to be relevant everywhere: Why are they using 'our' tag?

**Reply by ThatXliner** [reply]
Wait, the results of which tag?

**Reply by MegaIng** [reply]
The link you posted, e.g. `lark`

**Reply by ThatXliner** [reply]
All the results in https://github.com/topics/lark-parser relates to this project in one way or another. Some of the results in https://github.com/topics/lark are not relevant to this project
